pkg/cmd/dcos.go

```diff
@@ -0,0 +1,41 @@
+// Package cmd defines commands for the DC/OS CLI.
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/dcos/dcos-cli/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands.
+var rootCmd = &cobra.Command{
+	Use: "dcos",
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+type Cluster struct {
+	Config config.Config
+}
+
+// This is a temporary function until we have the proper abstraction for this.
+func attachedCluster() *Cluster {
+	path := "/home/bamarni/.dcos/dcos.toml"
```

Comment:
Let's replace this line by:
```
usr, _ := user.Current()
path := filepath.Join(usr.HomeDir, ".dcos", "dcos.toml")
```
This requires `"os/user"` and `"path/filepath"`.

Reply:
makes sense :+1: